bitnami/metallb/templates/speaker/daemonset.yaml

```diff
@@ -167,6 +205,114 @@ spec:
             {{- if .Values.speaker.extraVolumeMounts }}
             {{- include "common.tplvalues.render" (dict "value" .Values.speaker.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
+        {{- if .Values.speaker.frr.enabled }}
+        - name: frr
+          image: {{ include "common.images.image" (dict "imageRoot" .Values.speaker.frr.image "global" .Values.global) }}
+          imagePullPolicy: {{ .Values.speaker.frr.image.pullPolicy }}
+          {{- if .Values.speaker.frr.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.speaker.frr.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.speaker.frr.image.pullPolicy }}
+          imagePullPolicy: {{ .Values.speaker.frr.image.pullPolicy }}
+          {{- end }}
+          volumeMounts:
+            - name: frr-sockets
+              mountPath: /var/run/frr
+            - name: frr-conf
+              mountPath: /etc/frr
+          # The command is FRR's default entrypoint & waiting for the log file to appear and tailing it.
+          # If the log file isn't created in 60 seconds the tail fails and the container is restarted.
+          # This workaround is needed to have the frr logs as part of kubectl logs -c frr < speaker_pod_name >.
+          command:
+            - /bin/sh
+            - -c
+            - |
+              . /usr/lib/frr/frrcommon.sh
+              /usr/lib/frr/watchfrr $(daemon_list) &
+              attempts=0
+              until [[ -f /etc/frr/frr.log || $attempts -eq 60 ]]; do
```

Comment:
The script is not working as expected, here we've got the output:
```
/bin/sh: 4: [[: not found
/bin/sh: 4: 0: not found
2024/04/23 06:49:49 WATCHFRR: [T83RR-8SM5G] watchfrr 9.0.2 starting: vty@0
2024/04/23 06:49:49 WATCHFRR: [ZCJ3S-SPH5S] zebra state -> down : initial connection attempt failed
2024/04/23 06:49:49 WATCHFRR: [ZCJ3S-SPH5S] mgmtd state -> down : initial connection attempt failed
2024/04/23 06:49:49 WATCHFRR: [ZCJ3S-SPH5S] bgpd state -> down : initial connection attempt failed
2024/04/23 06:49:49 WATCHFRR: [ZCJ3S-SPH5S] staticd state -> down : initial connection attempt failed
2024/04/23 06:49:49 WATCHFRR: [ZCJ3S-SPH5S] bfdd state -> down : initial connection attempt failed
2024/04/23 06:49:49 WATCHFRR: [YFT0P-5Q5YX] Forked background command [pid 51]: /usr/lib/frr/watchfrr.sh restart all
[113|mgmtd] sending configuration
[114|zebra] sending configuration
[120|bgpd] sending configuration
[128|watchfrr] sending configuration
[114|zebra] done
[113|mgmtd] done
[130|staticd] sending configuration
[131|bfdd] sending configuration
2024/04/23 06:49:50 WATCHFRR: [VTVCM-Y2NW3] Configuration Read in Took: 00:00:00
Waiting for children to finish applying config...
[128|watchfrr] done
[120|bgpd] done
[130|staticd] done
[131|bfdd] done
2024/04/23 06:49:50 WATCHFRR: [QDG3Y-BY5TN] zebra state -> up : connect succeeded
2024/04/23 06:49:50 WATCHFRR: [QDG3Y-BY5TN] mgmtd state -> up : connect succeeded
2024/04/23 06:49:50 WATCHFRR: [QDG3Y-BY5TN] bgpd state -> up : connect succeeded
2024/04/23 06:49:50 WATCHFRR: [QDG3Y-BY5TN] staticd state -> up : connect succeeded
2024/04/23 06:49:50 WATCHFRR: [QDG3Y-BY5TN] bfdd state -> up : connect succeeded
2024/04/23 06:49:50 WATCHFRR: [KWE5Q-QNGFC] all daemons up, doing startup-complete notify
/bin/sh: 4: [[: not found
/bin/sh: 4: 1: not found
/bin/sh: 4: [[: not found
/bin/sh: 4: 2: not found
/bin/sh: 4: [[: not found
/bin/sh: 4: 3: not found
/bin/sh: 4: [[: not found
```

Reply:
I moved to `/bin/bash` instead.